internal/adapters/terraform/aws/ec2/vpc.go

```diff
@@ -14,12 +14,16 @@ type sgAdapter struct {
 	sgRuleIDs terraform.ResourceIDResolutions
 }
 
-func adaptDefaultVPCs(modules terraform.Modules) []ec2.DefaultVPC {
-	var defaultVPCs []ec2.DefaultVPC
+func adaptVPCs(modules terraform.Modules) []ec2.VPC {
+	var defaultVPCs []ec2.VPC
 	for _, module := range modules {
 		for _, resource := range module.GetResourcesByType("aws_default_vpc") {
-			defaultVPCs = append(defaultVPCs, ec2.DefaultVPC{
-				Metadata: resource.GetMetadata(),
+			defaultVPCs = append(defaultVPCs, ec2.VPC{
```

Comment:
As the base object was changed, all usage needs to be updated. 

We don't have a check for this for Terraform and Cloudformation at this time, but the values still need sensible values